mkdir: cannot create directory ‘/home/vagrant/.rbenv/versions/2.5.0’: Permission denied

## Versions
  - Vagrant: 2.2.4
  - Provider: Virtualbox 6.0.6
  - Homestead: 7.2.1

## Host operating system
Ubuntu 16.04 - Xenial

## Homestead.yaml
```yaml
---
# box: laravel/homestead-7
ip: "192.168.10.10"
memory: 2048
cpus: 1
provider: virtualbox
version: ">=0.5.0"

authorize: ~/.ssh/id_rsa.pub

keys:
    - ~/.ssh/id_rsa

folders:
    - map: ~/sites
      to: /home/vagrant/sites

sites:
    - map: cobaia.test
      to: /home/vagrant/sites/cobaia

databases:

variables:
    - key: 'APP_ENV'
      value: 'local'

ruby: true
```

## Vagrant destroy & up output
Everything happens OK

## Expected behavior
When running `rbenv install 2.5.0` it should install the ruby 2.5.0

## Actual behavior
```
Downloading ruby-2.5.0.tar.bz2...
-> https://cache.ruby-lang.org/pub/ruby/2.5/ruby-2.5.0.tar.bz2
Installing ruby-2.5.0...
mkdir: cannot create directory ‘/home/vagrant/.rbenv/versions/2.5.0’: Permission denied

BUILD FAILED (Ubuntu 18.04 using ruby-build 20190423)

Inspect or clean up the working tree at /tmp/ruby-build.20190423185518.5423
Results logged to /tmp/ruby-build.20190423185518.5423.log

Last 10 log lines:
/tmp/ruby-build.20190423185518.5423 ~
/tmp/ruby-build.20190423185518.5423/ruby-2.5.0 /tmp/ruby-build.20190423185518.5423 ~
```
## Steps to reproduce
1. vagrant up
2. vagrant ssh
3. rbenv install 2.5.0

--- a/scripts/install-ruby.sh
+++ b/scripts/install-ruby.sh
@@ -17,3 +17,4 @@ rbenv global 2.6.1
 apt-get -y install ruby`ruby -e 'puts RUBY_VERSION[/\d+\.\d+/]'`-dev
 gem install rails -v 5.2.2
 rbenv rehash
+chown -Rf vagrant:vagrant /home/vagrant/.rbenv

